views/recipe.pug

```diff
@@ -9,61 +9,66 @@ html(lang='en')
         //-<script src='https://kit.fontawesome.com/33df1dd2cb.js' crossorigin='anonymous'></script>
         title Recipes
     body
-        block content
-            div(class='header-1')
-                h1=recipe.title
-                h2(class='recipeRating')=recipe.avgRating
-                //- still need to get avgRating
-            div(class='recipe__recipeCard')
-                div(class='recipe__description')
-                    each recipeDescription in recipeDescriptions
-                        p=recipeDescription
-                    div
-                        img(src=recipe.foodImg, alt=recipe.title)
-                    div(class='recipe__ingredients')
-                        h3(class='header-3') Ingredients
-                        ul
-                            each recipeIngredient in recipeIngredients
-                                li=recipeIngredient
-                    div(class='recipe__directions')
-                        h3(class='header-3') Directions
-                        ul
-                            each recipeDirection in recipeDirections
-                                li=recipeDirection
-                        div(class='reviewCookbookWrap')
-                            div(class='recipe__reviewButton')
-                                button(id='displayReviewButton') Write A Review
-                            div
-                                button(class='recipe__cookBook--button') Add to Cookbook
-                                //- ask about this for add to cookbooks
+        div(class='header-1')
+            h1=recipe.title
+            h2(class='recipeRating')=recipe.avgRating
+            //- still need to get avgRating
+        div(class='recipe__recipeCard')
+            div(class='recipe__description')
+                each recipeDescription in recipeDescriptions
+                    p=recipeDescription
+                div
+                    img(src=recipe.foodImg, alt=recipe.title)
+                div(class='recipe__ingredients')
+                    h3(class='header-3') Ingredients
+                    ul
+                        each recipeIngredient in recipeIngredients
+                            li=recipeIngredient
+                div(class='recipe__directions')
+                    h3(class='header-3') Directions
+                    ul
+                        each recipeDirection in recipeDirections
+                            li=recipeDirection
+                    div(class='reviewCookbookWrap')
+                        div(class='recipe__reviewButton')
+                            button(id='displayReviewButton') Write A Review
+                        div
+                            button(class='recipe__cookBook--button') Add to Cookbook
+                            //- ask about this for add to cookbooks
```

Comment:
If you want to leave this as a TODO, please be sure you are referencing a specific github issue here as well.

Reply:
Issue is in progress